### ISSUE TITLE
Scan error compiling swift sources

### ISSUE THREAD

**Issue by leonardowistuba** [fastlane/fastlane#4092]
### New Issue Checklist

- [ x ] Updated fastlane to the latest version
- [ x ] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description
After I run scan using Quick/Nimble I've received this output when trying to compile Nimble sources
<img width="851" alt="screen shot 2016-04-06 at 7 45 15 pm" src="https://cloud.githubusercontent.com/assets/18118251/14335358/e9e57dd0-fc30-11e5-9399-25ccd750caa9.png">

and failing with scan/error_handler.rb:24:in `handle_build_error': [!] Error building the application - see the log above (RuntimeError)

Anyone know what is going on?

Thanks!

##### Complete output when running fastlane, including the stack trace and command used

```
https://gist.github.com/leonardowistuba/9e77a1f2e5de991d11bcfdd6413e9c21
```

fastlane version (fastlane -v): fastlane 1.80.0
Do you use bundler, rbenv or rvm? rvm
ruby: ruby-2.2.3

**Comment by hartbit** [comment]
Seeing how the errors are on a line with the new `#file` and `#fline` syntax of Swift 2.2, my best guess is that the build tools are not pointing to a Xcode 7.3 version of Xcode. Run the following to check:

```
xcode-select -p
```

**Comment by leonardowistuba** [comment]
Thanks, that's the problem. I'm using the xcode 7.2.1 version because my tests are failing with code 65 in xcode 7.3 too https://github.com/fastlane/fastlane/issues/3886

I will wait for a fix for XCode 7.3 :/

**Comment by TKBurner** [comment]
Hi @leonardowistuba. Thanks for following up. We are going to track cod 65 errors on #3886. I'm going to close this issue out as housekeeping. Let me know if you have any questions :+1:
